packages/manager/modules/pci/src/projects/project/project.html

```diff
@@ -295,6 +306,7 @@
     <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable">
         <div class="modal-content">
             <oui-modal
+                data-ng-show="$ctrl.onActivateDiscoveryModalDisplay()"
```

Comment:
**nitpick**:  this function does not exist, to remove. Please be careful

Reply:
Done.